garage/envs/dm_control/dm_control_env.py

```diff
@@ -39,13 +39,14 @@ def step(self, action):
         if time_step.reward:
             self._total_reward += time_step.reward
         return Step(
-            flatten_observation(time_step.observation), time_step.reward,
-            time_step.step_type == StepType.LAST, **time_step.observation)
+            flatten_observation(time_step.observation)['observations'],
```

Comment:
Although it's not enforced by the lint tools since the legacy code has a lot of mixed cases, better use double quotes with strings.